Const-corrected and updated README.md

AliLogic opened the pull request.

AliLogic committed:
const-corrected!

AliLogic committed:
Update pawn.json for compatibility with sampctl

AliLogic committed:
Update README.md

AliLogic committed:
Revert pawn.json

Southclaws commented on README.md:
```diff
@@ -76,3 +76,8 @@ public OnDialogPerformed(playerid, dialog[], response, success)
+
+## Credits
+Emmet_ - for easyDialog
+Southclaws - for samp-ctl
```
I hate being pedantic [but this is technically sampctl](https://github.com/Southclaws/sampctl/wiki/What-Does-sampctl-Mean)

AliLogic committed:
Fixed an oof

AliLogic committed:
fixed sampctl typo

AliLogic commented:
Users who don't have their a_samp const corrected will warnings, so if anybody has put up the const corrected default libraries, then put up a link here and i'll add it.

AliLogic committed:
Update easyDialog.inc

Changed "success" to be a bool.

AliLogic committed:
Update easyDialog.inc

AliLogic committed:
Update README.md

AliLogic commented:
I've fixed my oofs (errors) and added a warning that users should use sampctl's std lib. If there's anything that I should take care of, please let me know :) Have a nice day.

AliLogic committed:
Changed the s_DialogOpened to be a bool

Southclaws commented:
Thank you!
